apps/console/src/features/users/components/user-profile.tsx

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ * Copyright (c) 2021, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Is this intentional change?

Reply:
It's a typo. FIxed with c100ded3bd2bd6827df434b5029d1f865cd151e8